src/lib/dev-environment/dev-environment-core.js

```diff
@@ -148,25 +148,14 @@ export async function runWp( slug: string, args: Array<string> ) {
 		throw new Error( 'Environment not found.' );
 	}
 
-	await landoExec( instancePath, 'wp', args );
-}
-
-export async function runAddSite( slug: string, newSiteSlug: string, newSiteTitle: string ) {
-	debug( 'Will run a wp command on env', slug, 'for', newSiteSlug, ' - ', newSiteTitle );
-
-	const instancePath = getEnvironmentPath( slug );
-
-	debug( 'Instance path for', slug, 'is:', instancePath );
+	const command = args.shift();
 
-	const environmentExists = fs.existsSync( instancePath );
-
-	if ( ! environmentExists ) {
-		throw new Error( 'Environment not found.' );
+	let commandArgs = [ ...args ];
+	if ( 'add-site' === commad ) {
```

Comment:
```suggestion
	if ( 'add-site' === command ) {
```

Reply:
oh yeah also noticed that now :) Thanks. Should be fixed